src/multiplayer-servers/multiplayer-services/server-protection/protocols.md

```diff
@@ -0,0 +1,68 @@
+# Ports and Protocols
+
+Game server ports are often vulnerable to attacks. 
+GameFabric offers so-called Protocols to protect your ports with powerful SteelShield mitigations.
```

Comment:
`so-called` seems to work against us, as it is largely used sarcastically in every day speech.
```suggestion
GameFabric offers Protocols to protect your game server traffic with powerful SteelShield mitigations.
```